internal/rbd/controllerserver.go

```diff
@@ -262,7 +294,7 @@ func (cs *ControllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 	return &csi.CreateVolumeResponse{Volume: volume}, nil
 }
 
-func createBackingImage(ctx context.Context, cr *util.Credentials, rbdVol *rbdVolume, rbdSnap *rbdSnapshot) error {
+func (cs *ControllerServer) createBackingImage(ctx context.Context, cr *util.Credentials, rbdVol *rbdVolume, rbdSnap *rbdSnapshot) error {
```

Comment:
why make this a method for ControllerServer? it is not used in the function...

Reply:
This is required for locking

Reply:
but it is not used in this function? or maybe a follow up commit does that? It looks unneeded in https://github.com/ceph/ceph-csi/pull/1160/commits/b5209f8f74bde7b2a84f287c9b5c58e09e02a751